sublime/Toit.sublime-syntax

```diff
@@ -0,0 +1,179 @@
+%YAML 1.2
```

Comment:
We should try to use the vscode syntax highlighting here (as much as possible), so that updates are easier.
For now we can just use this version.